src/Fuel-Platform-Pharo-Core/FLPharoPlatform.class.st

```diff
@@ -39,13 +39,13 @@ FLPharoPlatform >> absolutePathOf: aString [
 	^ aString asFileReference fullName
 ]
 
-{ #category : #'accessing - compiler' }
+{ #category : #'accessing-compiler' }
```

Comment:
Please use "accessing - compiler" and not "accessing-compiler". Otherwise we have shadowed categories.

Reply:
But I wonder how this happens... there seems to be different code in the image than in the repository. I just commited, no changes done by me on the level of the categories

Reply:
Bad - I had similar trouble yesterday evening with the pull request for Microdown - Iceberg showed me more than I have actually changed (like Baseline or other).